lib/src/equatable.dart

```diff
@@ -43,7 +43,7 @@ abstract class Equatable {
   bool? get stringify => null;
 
   @override
-  bool operator ==(Object other) =>
+  bool operator ==(Object? other) =>
```

Comment:
why is this change needed in the case of `Equatable`? Since developers are in full control of this class I would prefer not to make this change unless there's a concrete use-case for it. Thoughts?